src/helm/benchmark/huggingface_registration.py

```diff
@@ -55,7 +53,9 @@ def register_huggingface_model(
                 description=helm_model_name,
                 access="open",
                 release_date=date.today(),
-                tags=[TEXT_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG],
+                modality=["text"],
+                functionality="full",
```

Comment:
We should probably do a design review with @percyliang and @teetone about how we want to represent this. This has implications on how "type checking" works (i.e. validating which scenarios work with which models).

An alternative proposal: instead of modality and functionality, we could have input types and output types, where input types can be "text", "messages", "multimedia", and output types can be "text", "text_with_logprobs", "multimedia", etc.

Reply:
Yeah, we should do a design doc for this first and think through all the different models that we might have (chat, speech, video, etc.).  It's unclear that we even want to keep this since I think we might want to manually select individual models to run anyway.

Also, whether a model returns logprobs is more a function of the deployment rather than the model.

Reply:
The design doc is now accessible here: https://docs.google.com/document/d/1Qofi6z8hST3CuiB47lUVrkJxYpAp3tNumwg0v3EHWXQ/edit